### PR TITLE
Add Alluxio native api for maxthroughput in stress bench

### PR DESCRIPTION
**Pull request by yyongycy**
This PR is adding an option to have MaxThroughput calling Alluxio Native API.
Stressbench(MaxThroughput) tool supports Alluxio HDFS API, the PR is going to add Alluxio Native API support.

This PR is based on https://github.com/Alluxio/alluxio/pull/14127

### CODE REVIEW

**Comment by yyongycy** [comment]
@jiacheliu3 @HelloHorizon @yuzhu  these changes are very similar to clientIO part.

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks for the improvement!

File: stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java [hunk 1]
```diff
@@ -236,7 +266,7 @@ public MasterBenchTaskResult runLocal() throws Exception {
 
     List<Callable<Void>> callables = new ArrayList<>(mParameters.mThreads);
     for (int i = 0; i < mParameters.mThreads; i++) {
-      callables.add(new BenchThread(context, mCachedFs[i % mCachedFs.length]));
+      callables.add(getBenchThread(context, i));
```
Comment (HelloHorizon) [inline stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java:269]: why we change the `mCachedFs` to `i`?

**Review by HelloHorizon — COMMENTED** [review]
@yyongycy Thanks for the improvement! Left one minor question.

File: stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java (hunk 1 above)
Comment (yyongycy) [reply stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java:269]: This is moving the new to a function, and pass index to the function

**Review by HelloHorizon — APPROVED** [review]
LGTM, also remember to update the document :)

**Review by yuzhu — APPROVED** [review]
Thanks for the improvement, LGTM

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**